src/components/container/DeviceDataMonthCharts/DeviceDataMonthCharts.stories.tsx

```diff
@@ -13,7 +13,7 @@ export default {
 } as ComponentMeta<typeof DeviceDataMonthCharts>;
 
 const Template: ComponentStory<typeof DeviceDataMonthCharts> = (args: DeviceDataMonthChartsProps) =>
-	<Layout>
+	<Layout  colorScheme="auto">
```

Comment:
Not a big deal, but looks like most (maybe all) of these have two spaces before the attribute.